easybuild/easyconfigs/p/PAGIT/PAGIT-V1.64bit.eb

```diff
@@ -0,0 +1,43 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Tarball'
+
+name = 'PAGIT'
+version = 'V1'
+versionsuffix = '.64bit'
+
+homepage = 'http://www.sanger.ac.uk/science/tools/pagit'
+description = """PAGIT addresses the need for software to generate high quality draft genomes"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['ftp://ftp.sanger.ac.uk/pub/resources/software/pagit/']
+sources = ['%(name)s.%(version)s%(versionsuffix)s.tgz']
+
+osdependencies = ['tcsh']
+
+modextrapaths = {
+       'PATH': "bin:%(installdir)s/PAGIT/bin:%(installdir)s/PAGIT/bin/pileup_v0.5/:%(installdir)s/PAGIT/bin/pileup_v0.5/ssaha2:%(installdir)s/PAGIT/IMAGE/:%(installdir)s/PAGIT/ABACAS:%(installdir)s/PAGIT/ICORN:%(installdir)s/PAGIT/RATT",
```

Comment:
@pescobar I this case, you should i) use a list rather than one big string with `:`, ii) not use `%(installdir)s` in the entries, that gets added automatically for each entry:

```python
'PATH': ['bin', 'PAGIT/bin', ...],
```